python/taichi/ui/gui.py

```diff
@@ -699,6 +735,31 @@ def arrow_field(self,
                                       2)
         self.arrows(base, direction, radius=radius, color=color, **kwargs)
 
+    def vector_field(self, vector_field, arrow_spacing=5, color=0xFFFFFF):
```

Comment:
ditto, use GGUI. Arrows there needs to be colored lines maybe...

Reply:
After briefly looking into the existing functions in `ui/canvas.py`, I think I will combine a call to `canvas.lines()` and a call to `canvas.triangles()` to make the arrows. Let me know if you have further comments!